File: app/src/main/java/com/kelsos/mbrc/annotations/Connection.java
package com.kelsos.mbrc.annotations;

import android.support.annotation.IntDef;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

public class Connection {
  public static final int OFF = 0;
  public static final int ON = 1;
  public static final int ACTIVE = 2;

  private Connection() {
    //no instance
  }

  @Retention(RetentionPolicy.SOURCE)
  @IntDef({
      OFF,
      ON,
      ACTIVE
  })
  public @interface Status {

  }
}

File: app/src/main/java/com/kelsos/mbrc/annotations/Connection.kt
package com.kelsos.mbrc.annotations

import androidx.annotation.IntDef

object Connection {
  const val OFF = 0
  const val ON = 1
  const val ACTIVE = 2

  @Retention(AnnotationRetention.SOURCE)
  @IntDef(OFF, ON, ACTIVE)
  annotation class Status
} // no instance
